<?xml version="1.0"?>
<launch>

	<node name="left_etherdaq_node" pkg="yumi_optodaq" type="left_etherdaq_node" respawn="false" output="screen" ns="yumi">
		<param name="ip" value="192.168.125.3"/>
	    <param name="pub_rate_hz" value="200.0"/>
	    <param name="filter" value="4"/>
	    <param name="frame_id" value="optodaq_l_sensor_link"/>
	    <param name="cancel_bias" value="false"/>
	</node>

</launch>
